test/Microsoft.AspNet.Server.KestrelTests/EngineTests.cs

```diff
@@ -54,6 +56,14 @@ ILibraryManager LibraryManager
             }
         }
 
+        ILoggerFactory LoggerFactory
+        {
+            get
```

Comment:
Sorry if I wasn't clearer in my earlier comment: I'm recommending that these helper properties be completely removed because they don't really add much value. They don't have state and their content is trivial. I recommend referencing `NullLoggerFactory.Instance` directly in the individual unit tests like this:

```c#
var engine = new KestrelEngine(LibraryManager, NullLoggerFactory.Instance);
```